src/python/foglamp/translators/omf_translator.py

```diff
@@ -650,56 +622,47 @@ def send_omf_message_to_end_point(message_type, omf_data):
 
     """
 
-    global _log
+    global _omf_max_retry
 
-    try:
-        msg_header = {'producertoken': _producer_token,
-                      'messagetype':   message_type,
-                      'action':        'create',
-                      'messageformat': 'JSON',
-                      'omfversion':    '1.0'}
+    sleep_time = _omf_retry_sleep_time
```

Comment:
_omf_retry_sleep_time is undefined variable. Its not defined and not initialised with a default value.